patches/0001-flatpak-Use-yarn-directly-with-offline-flag-in-post-.patch

```diff
@@ -0,0 +1,86 @@
+From e1301100ca199f100c4b67952a3d57ccf01a0aa9 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Dami=C3=A1n=20Nohales?= <damiannohales@gmail.com>
+Date: Fri, 6 Nov 2020 16:00:36 -0300
+Subject: [PATCH 1/2] flatpak: Use yarn directly with --offline flag in
+ post-install script
```

Comment:
I think this patch could be simplified by upstream supporting an `OFFLINE=1` environment variable check, aside from the `findYarnVersion` being removed (defaulting to calling the `yarn` on `PATH`).

@advaithm any context on why you're not calling the local `yarn` here?

Reply:
I believe we are, we build and install yarn in the previous step. during clean up all the files in the yarn step are removed.

Reply:
If it's related to this version then this is ahead of what we're using in the project: `1.22.1`

https://github.com/flathub/flathub/blob/98e64ec2ff048cda986ca2d0fd9c165078f4fc20/io.github.shiftey.Desktop.yaml#L33-L36

Reply:
what version should I downgrade too?

Reply:
The other reason why we have a local `yarn` in the project is to ensure contributors stay up to date, without necessarily needing to update their global `yarn`.

Reply:
Installing a more recent version globally is fine (it needs at least a global shim to work easily on your terminal), but dropping the lookup and use of the local yarn could lead to some subtle differences in behaviour if there are updates that affect installation and usage of packages in the project.

Reply:
I'll look into updating the patch to call the local version of yarn

Reply:
I've made a patch to use the local yarn version, testing now

Reply:
it works, moving to vendor yarn